var/spack/repos/builtin/packages/intel-mpi/package.py

```diff
@@ -83,7 +83,14 @@ def setup_dependent_package(self, module, dep_spec):
         else:
             raise RuntimeError('No suitable bindir found')
 
-        self.spec.mpicc = join_path(bindir, 'mpicc')
```

Comment:
Out of curiosity, does either the before or after work for anyone else? I installed `intel-mpi` with Spack and there were no MPI wrappers at all in `prefix.bin`. For me, they were in `prefix.compilers_and_libraries.linux.mpi.intel64.bin`. I fixed this in #4300.